File: 136Single Number.c
#include <stdio.h>

int singleNumber(int* nums, int numsSize) {
  int i = 0;int result = 0;
  for (i = 0;i<numsSize; i++){
    result^=nums[i];
  }
  return result;
}


int main(int argc, char const *argv[]) {
  int nums[] = {1,2,2,3,3};
  int a = singleNumber(nums, 5);
  printf("%d\n", a);
  return 0;
}

File: 136. Single Number.py
# Given an array of integers, every element appears twice except for one. Find that single one.
#
# Note:
# Your algorithm should have a linear runtime complexity. Could you implement it without using extra memory?
#
# Subscribe to see which companies asked this question
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        for i in range(len(nums)):
            result ^= nums[i]
        return result
